### DISCUSSION TITLE
Recommendation to Adopt Docusaurus for  django-rest-framework Documentation

### DISCUSSION THREAD

**Discussion by vicheanath** [encode/django-rest-framework#9106] [Ideas & Suggestions]

**Comment by vicheanath** [comment]
<img width="1511" alt="Screenshot 2023-09-12 at 5 14 52 PM" src="https://github.com/encode/django-rest-framework/assets/48352653/9293a91e-5f27-41f7-82ba-c823d6211310">

**Comment by shakori999** [comment]
that would be very nice!, if they approved i would like to contribute to this project.
